src/loaddb/load_server_loader.cpp

```diff
@@ -117,8 +117,28 @@ namespace cubload
     cubthread::entry &thread_ref = cubthread::get_entry ();
     LC_FIND_CLASSNAME found = LC_CLASSNAME_EXIST;
     LC_FIND_CLASSNAME found_again = LC_CLASSNAME_EXIST;
+    const char *dot = strchr (class_name, '.');
```

Comment:
Is there any other way to check? For example checking the flag

Reply:
It doesn't seem to be a problem with loaddb only.
I will think more through other issues.